requirements.txt

```diff
@@ -0,0 +1,13 @@
+black==22.3.0
+jupyter==1.0.0
+matplotlib==3.5.1
+numba==0.55.1
+numpy==1.21.5
+pandas==1.4.1
+pyarrow==7.0.0
+pyteomics==4.5.3
+scipy==1.8.0
+seaborn==0.11.2
+tqdm==4.64.0
+pathos
```

Comment:
For full reproducibility of such one-off analyses, pinning a specific version is recommended.

Reply:
Will do that